qt/aqt/browser/previewer.py

```diff
@@ -218,7 +220,14 @@ def _render_scheduled(self) -> None:
             txt = self.mw.prepare_card_text_for_display(txt)
             txt = gui_hooks.card_will_show(txt, c, f"preview{self._state.capitalize()}")
             self._last_state = self._state_and_mod()
-        self._web.eval(f"{func}({json.dumps(txt)},'{bodyclass}');")
+
+        js: str
```

Comment:
This doesn't do any harm, but for future reference, it's not necessary to pre-declare variables in Python.